scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -37,6 +39,15 @@ function prepare_vm {
 	    printf "\n kernel-${kernel_version} installation failed \n"
     fi
     yum -y install kernel-devel-${kernel_version}
+
+    yum -y install yum-utils
+    yumdownloader -y kernel-${kernel_version} kernel-devel-${kernel_version}
```

Comment:
--resolve should be used to address all the dependencies. Sometime errata kernel update will comes with firmware dependencies and it should be taken care.

Reply:
added --resolve to download packages with all dependencies